Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -171,12 +171,14 @@ function Initialize-DevelopmentEnvironment {
 }
 
 function Invoke-PluginFolderScript {
-    $settings = Get-DeploymentSettings
-    $pluginFolder = (Get-RepositoryResolvedPath "Plugin").Path
+    Invoke-Task -name $MyInvocation.MyCommand.Name -task {
+        $settings = Get-DeploymentSettings
+        $pluginFolder = (Get-RepositoryResolvedPath "Plugin").Path
```

Comment:
should this be using the plugin property from the appsettings.json?

Reply:
also it looks like `Invoke-PluginFolderScript` is duplicated both in initdev and the deployment scripts but was already implemented in the plugin.psm1 script as `Get-Plugins`. we should probably clean this up.

Reply:
created [ODS-4630](https://tracker.ed-fi.org/browse/ODS-4630) to address these issues